oshc/oshc/urls.py

```diff
@@ -4,4 +4,5 @@
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'', include('main.urls')),
+    url(r'^accounts/', include('registration.backends.simple.urls')),
```

Comment:
You're using the [simple backend](https://django-registration-redux.readthedocs.io/en/latest/simple-backend.html) and you've created the files (email related) for the [default backend](https://django-registration-redux.readthedocs.io/en/latest/default-backend.html).  You should include `registration.backends.default.urls`.